### ISSUE TITLE
Pi4 - qt.ini or config.ini to boot with multi window?

### ISSUE THREAD

**Issue by retropieuser** [mgba-emu/mgba#2275]
I've been messing around with booting up from RetroPie. It works and plays really well, plus there's so much to customize when it comes to controls which is great! 

So far there's a couple of things I wonder if there's answers I'm missing in the config.ini or qt.ini?

Are there any commands or options that I could utilise to boot the multiplayer second window at the program's startup? Also using startx or XINIT I'm wondering if there's any ideas anyone has if that will allow me to boot 2 windows at once that way? As I can start a second window while in either but it'll only be on top of the original window so there's no way to move them that I can spot.

(Granted this is more a linux type question than a mgba question)

**Comment by endrift** [comment]
This isn't a thing currently. It might make sense to add command line arguments for it, but not in the ini. Also, you might want to use a window manager if that's a problem.

**Comment by retropieuser** [comment]
That's fair, just thought I'd check as the configs have things like window position, width, height so I thought maybe there'd be an option setting for the other window. Anywho I'll investigate and see what I can come up with using the window manager programs, thanks again!

**Comment by Dimensional** [comment]
Retropie disables the windows managers, Wayland or xorg for example, because it uses its own to hook directly into screen rendering. You'll have to read into their documentation regarding that. It should be listed with changing the default boot options for Raspberry Pi OS.

**Comment by retropieuser** [comment]
> Retropie disables the windows managers, Wayland or xorg for example, because it uses its own to hook directly into screen rendering. You'll have to read into their documentation regarding that. It should be listed with changing the default boot options for Raspberry Pi OS.

Yeah I think you're right as Retropie used to come with pixel desktop installed which had x org as a dependent? Anyway I'll mess around have a read and see what I can come up with. I can play a linked 2 player game when in the desktop environment no problem, just seeing if there's a way to run some commands so it's setup like that when starting the program from RetroPie. If I figure it out will post back here
